### PR TITLE
Signup: launch G Suite upsell discount A/B test.

### PR DESCRIPTION
**Pull request by taggon**
This will launch G Suite upsell discount abtest. For more context, see p9jf6J-Hb-p2 and p8Eqe3-sq-p2

# How to test

_Note: The following steps should be performed outside of a store sandbox because the G Suite works only with real domains._

_Note2: Do not forget to cancel both the domain and the G Suite purchases soon afterwards. Or A8C will be charged for it._

_Note3: You may need a number of free credit for the testing._

1. Ensure that you're assigned to `gsuiteDiscount` abtest with `discount` variation.
2. Create a website on a paid plan with a custom domain.
3. Make a purchase.
4. You should be able to see will see the G Suite upsell dialog.

5. When you choose to add an email address, a coupon code should be automatically applied to the cart.

### CODE REVIEW

**Comment by jancavan** [comment]
@taggon For some reason, I'm unable to see the G Suite screen following the steps. I'm using credits to pay, but after the payment screen, I'm taken to my site.

> Note2: Do not forget to cancel both the domain and the G Suite purchases soon afterwards. Or A8C will be charged for it.

Also, how do I cancel the purchase?

**Comment by jancavan** [comment]
I see it now after trying to add a new Domain, but also without the discount:

screen shot 2018-07-26 at 11 30 26 am

Only plan purchases have the discount.

File: client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx [hunk 1]
```diff
@@ -5,29 +5,89 @@
...
+	};
+
+	renderPrice() {
 		const { translate } = this.props;
 
+		if ( this.props.showDiscount ) {
+			return (
+				<Fragment>
+					<del>
+						{ translate( '%(monthlyPrice)s per user /month', {
```
Comment (a8ci18n) [inline client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( '%(monthlyPrice)s per user / month' )` *ES Score: 23*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jancavan** [comment]
> Only plan purchases have the discount.

Sorry, nevermind, I managed to get it to display after a few tries.

> Also, how do I cancel the purchase?

I also canceled my subscriptions from SA.

----------

I've made updates in this changeset: https://github.com/Automattic/wp-calypso/pull/26342/commits/75db1488b9d8416c49ddbe1c01f33cca903ae479

### Changes made:
- Use the `Badge` component
- Make sure there isn't a line break before the badge
- Use `<div>`s  instead of `<h5>`s
- Use lists instead of `<h5>`s
- On mobile, made adjustments to spacing so pricing texts don't look too cramped (See image below)
- Use `/month` instead of `/ month`:

 "$2.50 per user / month" looks like an "or"? - ie "per user or month"?
cc @anneforbush Can you please confirm if this is correct?

screen shot 2018-07-26 at 1 01 32 pm

Mobile:
screen shot 2018-07-26 at 1 02 31 pm


@anneforbush Should we change the price to `$2.50`?

**Comment by jancavan** [comment]
> "$2.50 per user / month" looks like an "or"? - ie "per user or month"?

Added back space between `/` and `month` since it seems like it's generally understood what it means anyway.. and it looks cleaner: https://github.com/Automattic/wp-calypso/pull/26342/commits/12bc6bbd164385b2f3dd3744dbb9439221acfce4

screen shot 2018-07-26 at 2 29 35 pm

**Comment by anneforbush** [comment]
> @anneforbush Should we change the price to $2.50?
@jancavan reconfirming what we chatted about in Slack: I'm fine with changing to $2.50 but to be consistent let's change the crossed out price of $4.17 to $4.20 so we have nice numbers all around. :) 

cc @taggon

**Comment by taggon** [comment]
@fditrapani @jancavan I intentionally disabled the A/B test in My Site > Domains since I thought the test is only for the signup flow. Should we enable the test on the domains page as well?

**Comment by fditrapani** [comment]
> Should we enable the test on the domains page as well?

I don't see why not. @anneforbush any concerns with adding the discount to regular domain purchases too?

**Comment by anneforbush** [comment]
>  @anneforbush any concerns with adding the discount to regular domain purchases too?

@fditrapani @taggon just tagged you both on a p2 post to discuss this. :)

**Comment by taggon** [comment]
@anneforbush @jancavan I'm also fine with `$2.50` and `$4.20` monthly prices. But I don't figure out the formula to get `$2.50` monthly price from `$29` annual price. The prices shouldn't be hardcoded since we need to support other currencies.

I made some formulas, but none of these return `$4.20` and `$2.50`.

1. Monthly price = Annual price ÷ 12. Round off the numbers to two decimal places.
Current approach. We get `$.4.17`(from `$50` annual price)  and `$2.42`(from `$29`) monthly prices.

2. Monthly price = Annual price ÷ 12. Round off the numbers to one decimal place and add zero.
In this way, we will get `$4.20`and `$2.40`.

3. Monthly price = Annual price ÷ 10. Round off the numbers to one decimal place and add zero.
The monthly prices will be `$5.00` and `$2.90`.

**Comment by taggon** [comment]
I moved my comment to the P2 post( p8Eqe3-sq-p2 ) since it would be a better place to discuss about the prices.

**Comment by taggon** [comment]
Now zero decimals are trimmed.

**Comment by jancavan** [comment]
@taggon I tried to give this a final look, but after payment, it takes me to a blank screen.

I used the `calypso.live` link above and followed the step-by-step instructions. 

screenshot 2018-08-02 09 44 00

Anyone experiencing the same issue?

**Comment by fditrapani** [comment]
I'm running into this issue too. I tried it Calypso live first and then tested it locally and in both cases couldn't get passed checkout. Here's my console.

image

**Comment by jancavan** [comment]
I also tried pulling the branch and testing locally, but it's still a blank screen. I'm getting the same Console error as @fditrapani.

**Comment by taggon** [comment]
@fditrapani @jancavan Sorry, I missed something while I was pushing the latest fix! It should be working now. :)

**Comment by fditrapani** [comment]
Works for me now: 

image

**Pull request merged**